packages/celeste-vue/src/components/table/table-caption.vue

```diff
@@ -0,0 +1,24 @@
+<script setup lang="ts">
+import type { HTMLAttributes } from 'vue';
+import clsx from 'clsx';
+
+const props = defineProps<TableCaptionProps>();
+</script>
+
+<script lang="ts">
+export interface TableCaptionProps {
+  class?: HTMLAttributes['class'];
+}
+</script>
+
+<template>
+  <caption :class="clsx('celeste-table-caption', props.class)">
+    <slot />
+  </caption>
+</template>
+
+<style scoped>
+.celeste-table-caption {
+  font: var(--paragraph-sm);
```

Comment:
thoughts on this polishing
```suggestion
  padding-block: var(--spacing-16);
  color: var(--color-text-soft-400);
  font: var(--paragraph-sm);
```